detect.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
this is not the version on the pi, it lacks the wantedby line.

Reply:
fuuu